src/general/Mutations.cs

```diff
@@ -142,8 +143,9 @@ public MicrobeSpecies CreateRandomSpecies(MicrobeSpecies mutated, int steps = 5)
 
         GameWorld.SetInitialSpeciesProperties(temp);
 
-        // TODO: in the old code GenerateNameSection was used to
-        // override the default species name here
+        var nameGenerator = SimulationParameters.Instance.NameGenerator;
```

Comment:
Could you add a comment like `// Override the default starting name to have more variability in the names` here?